scripts/package/package-debian.sh

```diff
@@ -97,28 +97,43 @@ create_debian_package(){
 test_debian_package(){
     header "Testing debian package"
     
-    # Set LAST_VERSION_URL to enable upgrade tests
-    export LAST_VERSION_URL="https://dotnetcli.blob.core.windows.net/dotnet/$CHANNEL/Installers/Latest/dotnet-ubuntu-x64.latest.deb"
+    install_bats
```

Comment:
As far as the package_tool goes, these tests are dropped by the tool itself next to the deb package, and are not at all specific to our package. For that reason I think leaving that functionality makes sense for the package_tool, if it ever had additional consumers.

We could have a separate implementation in C# which would be independent to our repo. I think this makes sense and will bring consistency.